RELEASE/scripts/autoscend/paths/legacy_of_loathing.ash

```diff
@@ -38,12 +38,18 @@ boolean lol_buyReplicas()
 		if(contains_text(page, "2002")) //2023
 		{
 			buy($coinmaster[Replica Mr. Store], 1, $item[Replica 2002 Mr. Store Catalog]);
+			auto_buyFrom2002MrStore();
 		}
 		if(contains_text(page, "patriotic eagle") && !is100FamRun()) //If this isn't a 100% familiar run, go ahead and get another familiar
 		{	
 			buy($coinmaster[Replica Mr. Store], 1, $item[replica sleeping patriotic eagle]);
 			use(1, $item[replica sleeping patriotic eagle]); // put in terrarium
 		}
+		if(contains_text(page, "august")) //2023
```

Comment:
is august sufficient? Maybe update to `august scepter` but minor and up to you

Reply:
I was mirroring what was on 2002, but specificity doesn't hurt